service_mesh/service_mesh_install/updating-ossm.adoc

```diff
@@ -4,42 +4,9 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-Updating {ProductName} requires extra steps before you update {product-title} to version 4.2. 
-You must upgrade {ProductName} to 1.0.2 before upgrading {product-title} 4.1.x to 4.2.
+If you selected the automatic update stream, updating {ProductName} doesn't require any extra steps. 
 
-.Prerequisites
+If you choose to update manually, the Operator Lifecycle Manager (OLM) controls the installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. It is deployed by default in {product-title}.
+OLM uses CatalogSources, which use the Operator Registry API, to query for available Operators as well as upgrades for installed Operators.
```

Comment:
"It" is ambiguous here, I'd change it to "OLM runs by default in {product-title}.

Reply:
@JStickler Good catch. Totally forgot about the title when I copied it from the old file. Made those changes. Should be ready.